BL scraper fails

[Failed run](https://github.com/openZH/covid_19/runs/555587928?check_suite_focus=true)

```
Run the scraper...
Traceback (most recent call last):
  File "/home/runner/work/covid_19/covid_19/scrapers/scrape_bl.sh", line 51, in <module>
    assert d, "Can't find a data table"
AssertionError: Can't find a data table
Error: Not matched input line: BL -                      -       - FAILED 2020-04-02T16:18:14+00:00  
# URLs: https://www.baselland.ch/politik-und-behorden/direktionen/volkswirtschafts-und-gesundheitsdirektion/amt-fur-gesundheit/medizinische-dienste/kantonsarztlicher-dienst/aktuelles/covid-19-faelle-kanton-basel-landschaft, https://www.statistik.bl.ch/files/sites/Grafiken/COVID19/20200402_COVID19_BL.htm 
# Errors: Missing date. Missing cases
```

--- a/scrapers/scrape_bl.sh
+++ b/scrapers/scrape_bl.sh
@@ -36,7 +36,7 @@ Datum, Bestätigte Fälle, Verstorbene
 # 2020-04-01
 """
 <pre id="data" style="display:none;">
-Datum, Bestätigte Fälle, Geheilte geschätzt, Verstorbene
+Datum, Bestätigte Fälle, Geheilte kalkuliert, Verstorbene
 28-02-2020,1,,
 29-02-2020,2,,
 ...
@@ -47,7 +47,7 @@ Datum, Bestätigte Fälle, Geheilte geschätzt, Verstorbene
 
 d = d.replace('\n', ' ')
 # Extract last line. Use non-greedy matching.
-d = sc.find(r'<pre id="data".*?> ?Datum, Bestätigte Fälle, Geheilte geschätzt, Verstorbene.*? ([^ ]+) ?</pre>', d)
+d = sc.find(r'<pre id="data".*?> ?Datum, Bestätigte Fälle, Geheilte kalkuliert, Verstorbene.*? ([^ ]+) ?</pre>', d)
 assert d, "Can't find a data table"
 
 c = d.split(',')

